bitnami/tomcat/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: tomcat
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/tomcat
   - http://tomcat.apache.org
-version: 10.4.3
+version: 10.4.4
```

Comment:
There is a version conflict. Updates the version taking into account the current version of master.